### PR TITLE
List enabled/disabled features in readme.

### PR DESCRIPTION
**Pull request by smarterclayton**
@pweil- this is the more detailed list, still not complete but please PTAL

### CODE REVIEW

**Comment by smarterclayton** [comment]
@danmcp as well

**Comment by derekwaynecarr** [comment]
This is really good, my only thought is maybe it should be a dedicated page linked from README

File: README.md [hunk 1]
```diff
@@ -140,6 +140,60 @@ FAQ
...
+  * These features are only enabled for cluster admins by default and we do not recommend enabling them for untrusted users
+  * We generally try to identify and fix these within 1 release of their availability
+* Tech Preview
+  * Features that are considered unsupported for various reasons are known as 'tech preview' in our documentation
+  * Kubernetes Alpha and Beta features are considered tech preview, although occasionally some features will be graduated early
+  * Any tech preview feature is not supported in OpenShift Container Platform except through exemption
+* Disabled Pending Migration
+  * These are features that are new in Kubernetes but which originated in OpenShift, and thus need migrations for existing users
+  * We generally try to minimize the impact of features introduced upstream to Kubernetes on OpenShift users by providing seamless
+    migration for existing clusters.
```
Comment (danmcp) [inline README.md]: clean up formatting

**Comment by danmcp** [comment]
+1

File: README.md (hunk 1 above)
Comment (smarterclayton) [inline README.md]: Isn't this correct?

**Comment by smarterclayton** [comment]
A dedicated page is nice but no one ever finds it.  I think this would be something in the official docs, in which case we'd describe some stuff here and kick it out.  I think anyone reading this readme wants the answer to this question, which is why it belongs here.

**Comment by smarterclayton** [comment]
Any comments on the content?

File: README.md [hunk 2]
```diff
@@ -140,6 +140,60 @@ FAQ
...
+Kubernetes Definitions:
+
+* Alpha
+  * The feature is available, but no guarantees are made about backwards compatibility or whether data is preserved when feature moves to Beta
+  * The feature may have significant bugs and is suitable for testing and prototyping
+  * The feature may be replaced or significantly redesigned in the future
+  * No migration is generally available to Beta
+* Beta
+  * The feature is available and generally agreed to solve the desired solution, but may need stabilization or additional feedback
+  * The feature is potentially suitable for limited production use, but
```
Comment (pweil-) [inline README.md]: remove ", but" since it doesn't lead in to the next sentence

File: README.md [hunk 3]
```diff
@@ -140,6 +140,60 @@ FAQ
...
+    migration for existing clusters.
+  * Generally these are addressed within 1 Kubernetes release
+
+The list of features that qualify under these labels is described below, along with additional context for why.
+
+Feature | Kubernetes | OpenShift | Justification
+------- | ---------- | --------- | -------------
+Third Party Resources | Alpha (1.3) | Not Yet Secure (1.2, 1.3) | Third party resources are still under active development.<br>Known issues include failure to clean up resources in etcd, which may result in a denial of service attack against the cluster.
+Garbage Collection | Alpha (1.3) | Not Yet Secure (1.3) | It is possible for a malicious user to trick another user into deleting a sensitive resource (like a quota or limit resource) during deletion.
+Pet Sets | Alpha (1.3) | Tech Preview (1.3) | PetSets are still being heavily iterated and no backwards compatibility is guaranteed. Also, Pet Sets allow users to create PVCs directly, and no controls exist to limit that.
```
Comment (pweil-) [inline README.md]: Mix of `Pet Sets` and `PetSets`

**Comment by pweil-** [comment]
content LGTM.  Closing https://github.com/openshift/origin/pull/9980 in favor of this PR

**Comment by smarterclayton** [comment]
Going to merge now, take feedback for other features, then move into formal docs, and then come back and update this page to link out.

[merge]

**Pull request merged**